api/scopes/views.py

```diff
@@ -59,11 +60,7 @@ class ScopeList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin):
     ordering = ('id', )  # default ordering
 
     def get_default_queryset(self):
-        scopes = []
-        for key, value in public_scopes.items():
-            if value.is_public:
-                scopes.append(Scope(id=key, scope=value))
-        return scopes
+        return ApiOAuth2Scope.objects.filter(is_public=True)
```

Comment:
Pulling scopes from db instead of hardcoded public scopes dictionary.  We need to make sure that prod db has all public scopes loaded in this table.